2-core-rules.md

```diff
@@ -214,108 +209,97 @@ _Fleeing_ models must take a _leadership test_. If the test is passed, the model
+Sometimes a charging warrior may not reach the enemy, usually due to an item use or a failed detection test. If this happens move your warrior up to 4" towards the enemy, stopping no closer than 1" away from them. This is called a failed charge. Failing a charge ends the model's turn, they may not shoot or cast spells that turn.
 
-Models cannot be moved into hand-to-hand combat except by charging – any move that brings a warrior into hand-to-hand combat is a charge by definition. A model that charges will 'strike first' in the first round of the ensuing combat.
+Models cannot be moved into hand-to-hand combat except by charging - any move that brings a warrior into hand-to-hand combat is a charge by definition. A model that charges will 'strike first' in the first round of the ensuing combat.
 
 #### Charging more than one opponent
 
 If you can move your warrior into base contact with more than one enemy model with its charge move, it can charge them both. This might be inadvisable as it'll then be fighting two enemies at once!
 
 #### Hiding
 
-The Hiding rule represents warriors concealing themselves in a way that our unmoving and dramatically posed models cannot. A hiding warrior keeps as still as possible, just peeking out of cover. 
-
-A model can hide if he ends his move behind a low wall, a column or in a similar position where he could reasonably conceal himself. The player must declare that the warrior is hiding and place a Hidden counter beside the model for it to count as being hidden. 
+The _Hiding_ rule represents warriors concealing themselves in a way that our unmoving and dramatically posed models cannot. A _Hiding_ warrior conceals themself as best they can by stiffening alongside a column, crouching behind a small wall, or hiding inside a barrel. A model can hide if they end their move behind a low wall, a column or in a similar position where they could reasonably conceal themself. The player must declare that the warrior is hiding for it to count as being _Hidden_. It may be worth placing a counter next to it so that other players are aware of it. 
 
 \page
 
-A model that runs, flees, is _stunned_ or charges cannot hide that turn. His sudden burst of speed does not give him time to hide.
+A model that is _Hidden_ is disregarded by the enemy warband's mandatory decisions. For example, a _Hidden_ model will not prevent an enemy warband from running, will not count as the closest model for the purpose of shooting, and _Frenzied_ models will not have to charge it. It is also considered wholly out of line of sight if an opponent wishes to charge them.
+
+A model that runs, is _incapacitated_, or charges cannot hide that turn. Their sudden burst of speed does not give them time to hide. They also may not enter the _Hidden_ mode if there are enemies within 4", or if they are within unobstructed line of sight to an enemy.
```

Comment:
```suggestion
A model that runs, is _incapacitated_, or charges cannot hide that turn. Their sudden burst of speed does not give them time to hide. They may not become _Hidden_ if there are enemies within 4", or if they are within unobstructed line of sight to an enemy.
```

Also, the nature of obstruction is unclear here.

Reply:
Agreed, though I found it difficult to concisely articulate the circumstances where someone could Hide without diagrams. Basically you can't go into hide if you're completely in the open to at least one enemy. (no more "I'm only hiding from x")